lib/druid/filter.rb

```diff
@@ -299,7 +307,35 @@ def initialize(dimension, bounds)
       @bound = {
         type: 'radius',
         coords: bounds.first,
-        radius: bounds.last,
+        radius: bounds.last
+      }
+    end
+  end
+
+  class BoundFilter < Filter
+    include BooleanOperators
+
+    def initialize(dimension, params)
+      super()
+      @type = 'bound'
+      @dimension = dimension
+      @ordering = params[:ordering]
+      @upper = params[:upper]
+      @upperStrict = params[:upperStrict]
```

Comment:
can you explain upperStrict.

Reply:
Its way to tell if we need `<` instead of `<=` https://druid.apache.org/docs/latest/querying/filters.html